src/metabase/api/session.clj

```diff
@@ -205,6 +209,36 @@
   (t2/delete! Session :id metabase-session-id)
   (mw.session/clear-session-cookie api/generic-204-no-content))
 
+(def metabase-slo-redirect-url
+  "The url that the IdP should respond to. Not all IdPs support this, but it's a good idea to send it just in case."
+  "/auth/sso/handle_slo")
+
+(defenterprise sso-info
+  "Returns SSO information, iff ee is enabled."
+  metabase-enterprise.sso.integrations.sso-settings
+  []
+  {})
+
+;; client initiates slo:
+(api/defendpoint POST "/logout"
```

Comment:
Why can't this endpoint be defined in the EE code, like in `metabase-enterprise.sso.api.sso` or something?